launcher/ui/pages/instance/ModFolderPage.cpp

```diff
@@ -195,8 +203,15 @@ void ModFolderPage::updateMods()
 
     auto mods_list = m_model->selectedMods(selection);
     bool use_all = mods_list.empty();
-    if (use_all)
+    if (use_all) {
         mods_list = m_model->allMods();
+        auto tempModList = mods_list;
+        for(auto mod : tempModList) {
+            if (mod->metadata()->do_updates==false) {
```

Comment:
this will segfault when the mod doesn't yet have a metadata (you should check `mod->metadata()` before accessing it)

Reply:
Not sure if I should bother fixing this as I'm guessing this would have to be redone if we want to store this information in some other way.